src/entity/discid.rs

```diff
@@ -0,0 +1,24 @@
+use serde::{Serialize, Deserialize};
+use crate::entity::release::Release;
+
+/// Disc ID is the code number which MusicBrainz uses to link a physical CD to a release listing.
+#[derive(Debug, Serialize, Deserialize, PartialEq, Clone)]
+#[serde(rename_all(deserialize = "kebab-case"))]
+pub struct Discid {
+    /// See [MusicBrainz Identifier](https://musicbrainz.org/doc/MusicBrainz_Identifier).
+    pub id: Option<String>,
```

Comment:
The id should be a `String`. It's always returned.
I'm not sure for the other ones, and I'm waiting for a reply on the IRC. Could always change them myself later